File: QTD2.Domain/Entities/Core/ClientSettings_LabelReplacement.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

using QTD2.Domain.Entities.Common;

namespace QTD2.Domain.Entities.Core
{
    public class ClientSettings_LabelReplacement : Entity
    {
        public string DefaultLabel { get; set; }
        public string LabelReplacement { get; set; }

        public void SetLabelReplacementText(string labelReplacement)
        {
            this.LabelReplacement = labelReplacement;
        }
    }
}

File: QTD.SPA/src/app/_DtoModels/ClientSettingsLabelReplacement/ClientSettings_LabelReplacement.ts
import { Entity } from "../Entity";

export class ClientSettings_LabelReplacement extends Entity {
  defaultLabel!: string;
  labelReplacement!: string;

  constructor(defaultLabel: string, labelReplacement: string){
    super();
    this.defaultLabel = defaultLabel;
    this.labelReplacement = labelReplacement
  }
}
